common/main/java/com/couchbase/lite/Fleece.java

```diff
@@ -71,7 +71,7 @@ static Object toCBLObject(@Nullable Object value) {
 
         throw new IllegalArgumentException(
             Log.formatStandardMessage(
```

Comment:
I've accepted the rest of this... but this looks wrong to me.  What is your reasoning?